frontend/src/lib/components/LemonTextArea/LemonTextArea.tsx

```diff
@@ -63,13 +69,96 @@ interface LemonTextMarkdownProps {
 }
 
 export function LemonTextMarkdown({ value, onChange, ...editAreaProps }: LemonTextMarkdownProps): JSX.Element {
+    const { objectStorageAvailable } = useValues(preflightLogic)
+
+    const [drag, setDrag] = React.useState(false)
+    const [isUploading, setIsUploading] = React.useState(false)
```

Comment:
Why `useEffect`, but `React.useState`?

Reply:
* updates front end coding conventions

![side-eye-monkey](https://user-images.githubusercontent.com/984817/195589755-2bfe7806-8267-4935-9453-fe2374cc2fae.png)

Reply:
Only the first 64 bits of a UUIDT are incremented-ish (the first 48 are the timestamp for smoother indexing in DBs), the other 64 bits are securely random – so this should be unguessable.

Reply:
So if you get the timestamp right, you have 1,208,925,819,614,629,273,600,000 guesses to get all possible files. Probably fine then.